12-hooks-in-react/src/components/Search.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect } from "react";
+
+const Search = () => {
+  const [term, setTerm] = useState("");
+
+  useEffect(()=>{
```

Comment:
I know this is practice but we also need to practice cleanliness, remove this and remove the useEffect from the import since there's no use for this

Reply:
Okay, will do. I will put back the useEffect when it has a proper functionality.